pkg/storage/storage.go

```diff
@@ -167,6 +165,8 @@ func GetPlatformStorage(listers *regopclient.Listers) (imageregistryv1.ImageRegi
 	// EmptyDir storage.
 	case configapiv1.LibvirtPlatformType:
 		fallthrough
+	case configapiv1.BareMetalPlatformType:
+		fallthrough
```

Comment:
The comment on L166 needs an update.  Or possible add a comment here explaining the justification for why we use EmptyDir on initial baremetal IPI install.